src/editor/UI/EditorView.re

```diff
@@ -30,5 +30,18 @@ let createElement = (~state: State.t, ~children as _, ()) =>
     let style =
       editorViewStyle(theme.colors.background, theme.colors.foreground);
 
-    (hooks, <View style> <EditorLayoutView state /> </View>);
+    if (state.zenMode) {
+      (
+        hooks,
+        <View style>
+          <EditorGroupView
+            state
+            windowId=0
```

Comment:
Not sure if this is correct?

Reply:
Ah, we should be setting it to the `state.windowManager.activeWindowId` in this case.

It's unlikely to cause problems at the moment, since it is only used for the mouse down handler (to decide which window to focus) - but it'd be good to have the right ID hooked up to prevent issues in the future.